src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -178,10 +245,28 @@ private CommandResult executeCommand(String commandText) throws CommandException
             logger.info("Result: " + commandResult.getFeedbackToUser());
             resultDisplay.setFeedbackToUser(commandResult.getFeedbackToUser());
 
-            if (commandResult.isShowHelp()) {
+            if (commandResult.isShowHelp() && commandResult.getFeedbackToUser()
```

Comment:
I know you split the if statement condition into two lines but the condition is still rather verbose and might warrant an abstraction into a boolean variable

Reply:
yes it is much neater after abstracting this